.github/workflows/release.yml

```
@@ -0,0 +1,194 @@
+            echo "## Changes" > release_notes.md
+            echo "" >> release_notes.md
+            echo "See full changelog at [CHANGELOG.md](CHANGELOG.md)" >> release_notes.md
+          fi
+        else
+          echo "## Release ${{ steps.tag.outputs.tag }}" > release_notes.md
+          echo "" >> release_notes.md
+          echo "This release includes various improvements and bug fixes." >> release_notes.md
+        fi
+
+    - name: Create GitHub Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)